### PR TITLE
Mn 2016 11 remove comment list

### LINKED ISSUE #462: Comments/documents of private projects are available using the API
Private content is not properly filtered, resulting in the following issue:

 - unauthenticated users can request all comments using the API
 - authenticated users can request all documents using the API

### PR DESCRIPTION
**Pull request by MagdaN**
fixes #462

### CODE REVIEW

File: euth/comments/templatetags/react_comments.py [hunk 1]
```diff
@@ -13,6 +15,10 @@
 def react_comments(context, obj):
     request = context['request']
 
+    serializer = ThreadSerializer(
+        obj.comments.all(), many=True, context={'request': request})
```
Comment (MagdaN) [inline euth/comments/templatetags/react_comments.py]: @slomo can we assume that everybody who wants to use the comments adds a comments generic relation to their model?

Comment (slomo) [inline euth/comments/templatetags/react_comments.py]: I think that wouldn't be to wired to request. If it turns out to be a problem at some point, than we can add an if here.

File: euth/documents/api.py [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
 
 from .models import Document
 from .serializers import DocumentSerializer
 
 
-class DocumentViewSet(viewsets.ModelViewSet):
+class DocumentViewSet(mixins.CreateModelMixin,
+                      mixins.RetrieveModelMixin,
+                      mixins.UpdateModelMixin,
+                      mixins.DestroyModelMixin,
```
Comment (MagdaN) [inline euth/documents/api.py]: Yes, you are right. It should be ok to remove them.

**Comment by MagdaN** [comment]
@slomo I removed retrieve and destroy

**Pull request merged**